tripy/tests/frontend/test_utils.py

```diff
@@ -291,3 +292,17 @@ def test_sync_arg_type_invalid(self):
             match=r"At least one of the arguments: \('a', 'b', 'c'\) must be a \`tripy.Tensor\`.",
         ):
             x, y, z = sync_arg_types(3.0, 3, 4)
+
+    def test_invalid_argument_type(self):
+        with helper.raises(
+            tp.TripyException,
+            match=r"convert_inputs_to_tensors decorator supports conversion only for Python numbers or sequences thereof",
```

Comment:
The user shouldn't need to know anything about `convert_inputs_to_tensors`. Can we just pass the argument through to the underlying API? Because of the function registry, each API does type checking, so it should fail with a more obvious error message

Reply:
Good point, I'll change the message

Reply:
No longer applicable, see below